src/wikimedia/wikiFetch.ts

```
@@ -67,10 +86,11 @@ export async function wikiFetchGet(wiki: PapyrusWiki, path: `/${string}`): Promi
     return await wikiFetchGetInternalCreatedPromise;
 }
 
-async function wikiFetchGetInternalWithParseJsonAndHandleErrors(wiki: PapyrusWiki, path: `/${string}`, retriesSoFar: number, url: URL): Promise<{}|null> {
+async function wikiFetchGetInternalWithParseJsonAndHandleErrors(wiki: PapyrusWiki, path: `/${string}`, retriesSoFar: number, url: URL): Promise<{}|null|typeof WIKI_FETCH_403FORBIDDEN> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)